sdk/cosmosdb/cosmos/src/retry/timeoutFailoverRetryPolicy.ts

```diff
@@ -77,6 +77,7 @@ export class TimeoutFailoverRetryPolicy implements RetryPolicy {
     }
     this.failoverRetryCount++;
     retryContext.retryCount++;
+    retryContext.retryLocationIndex = this.failoverRetryCount;
```

Comment:
failoverRetryCount can go up to 120, how is the retryLocationIndex account for that? Should it be using `%` as an operator to make sure it only

Reply:
@ealsur sorry for the confusion at that point I was still addressing your comments.
Now I have addressed all of them can you check it once.
Thanks.